vmngclient/session.py

```diff
@@ -189,6 +195,17 @@ def request(self, method, url, *args, **kwargs) -> vManageResponse:
         if response.request.url and "passwordReset.html" in response.request.url:
             raise InvalidOperationError("Password must be changed to use this session.")
 
+        if self.enable_relogin and response.headers.get("set-cookie"):
+            if not self.__second_relogin_try:
+                self.logger.warning("Relogin session...")
+                self.auth = vManageAuth(self.base_url, self.username, self.password, verify=False)
+                self.__second_relogin_try = True
+                return self.request(method, url, *args, **kwargs)
+            else:
+                self.logger.warning("Relogin failed.")
+        else:
+            self.__second_relogin_try = False
+
```

Comment:
Shouldn't we keep connection through an `auth` object instead of creating new one?